src/App/pages/ProductDetail/ProductDetail.tsx

```diff
@@ -49,14 +29,23 @@ const ProductDetail = () => {
         Back
       </a>
 
-      {productData && ( //не следует показывать карточку, если нет данных
+      {productDetailStore.productData && ( //не следует показывать карточку, если нет данных
         <>
           <div className={styles.cardInfo}>
-            <img src={productData?.images[0]} alt={productData?.title} />
+            <div className={styles.cardImg}>
+              <Carousel
+                images={productDetailStore.productData?.images}
+                alt={productDetailStore.productData?.title}
```

Comment:
```suggestion
                alt={productDetailStore.productData.title}
```
ты же выше проверил и productData точно есть